platform/src/main/java/org/stellar/anchor/platform/rpc/GetTransactionHandler.java

```diff
@@ -0,0 +1,94 @@
+package org.stellar.anchor.platform.rpc;
+
+import static org.stellar.anchor.api.rpc.method.RpcMethod.GET_TRANSACTION;
+import static org.stellar.anchor.platform.utils.PlatformTransactionHelper.toGetTransactionResponse;
+import static org.stellar.anchor.util.StringHelper.isEmpty;
+
+import java.util.Set;
+import org.stellar.anchor.api.exception.AnchorException;
+import org.stellar.anchor.api.exception.rpc.InvalidParamsException;
+import org.stellar.anchor.api.exception.rpc.InvalidRequestException;
+import org.stellar.anchor.api.platform.PlatformTransactionData.Sep;
+import org.stellar.anchor.api.rpc.method.GetTransactionRpcRequest;
+import org.stellar.anchor.api.rpc.method.RpcMethod;
+import org.stellar.anchor.api.sep.SepTransactionStatus;
+import org.stellar.anchor.asset.AssetService;
+import org.stellar.anchor.event.EventService;
+import org.stellar.anchor.metrics.MetricsService;
+import org.stellar.anchor.platform.data.JdbcSepTransaction;
+import org.stellar.anchor.platform.validator.RequestValidator;
+import org.stellar.anchor.sep24.Sep24TransactionStore;
+import org.stellar.anchor.sep31.Sep31TransactionStore;
+import org.stellar.anchor.sep6.Sep6TransactionStore;
+import org.stellar.anchor.util.Log;
+import org.stellar.anchor.util.SepHelper;
+
+public class GetTransactionHandler extends RpcMethodHandler<GetTransactionRpcRequest> {
```

Comment:
What do you think of making `RpcMethodHandler` more generic so that it only has the `handle` and `getRpcMethod`? We can create a subclass of that called something like `TransactionRpcMethodHandler` with `getNextStatus`, `getSupportedStatuses`, `getSupportedStatuses`, and `updateTransactionWithRpcRequest` that the existing RPCs can extend.

Reply:
It will make the structure a little bit more clear bit not much since functions related to status update (`getNextStatus`, `getSupportedStatuses`, `getSupportedStatuses`, `updateTransactionWithRpcRequest`, `isErrorStatus` and `isFinalStatus`) are embedded in the `handle` function in `RpcMethodHandler` class. 

Eventually everything in `RpcMethodHandler` will be moved to the subclass and leave `RpcMethodHandler` an empty parent class.